### ISSUE TITLE
The correct Delta_nu

### ISSUE THREAD

**Issue by Harry-Westwood** [Harry-Westwood/Y4-Project-InterNeuralStellar#48]
@grd349 

I've gotten the correct delta_nu column from Tanda but looking at the values 34.5% are 'nan'. Should I still progress with creating the grid with the correct delta_nu (using the suggested delta_nu bounds) and just exclude all of the nans despite this reducing the number of grid points substantially?

**Comment by grd349** [comment]
Hi @Harry-Westwood,

Good question - they are probably all models where we don't expect solar-like oscillations.  If that doesn't look like it is true then I would ask Tanda what is going on :)

**Comment by Harry-Westwood** [comment]
@grd349 

Tanda says that he only did the correct delta_nu calculations for every other point in the post main sequence. A 50% decrease in the training data is fairly significant, i could add in the rows that don't have the correct delta_nu by interpolating them if you think that is sensible?

**Comment by grd349** [comment]
> i could add in the rows that don't have the correct delta_nu by
> interpolating them if you think that is sensible?


Only if it is not too much work! :). How bad would it be to try without the
interpolation and then if that doesn't work look at the interpolation?

**Comment by Harry-Westwood** [comment]
@grd349 
It shouldn't be too much work, but I will leave it as an option to fall back on if the current grid I've made gives poor results.

**Comment by grd349** [comment]
Super! :)


*Dr Guy R. Davies*
PI - ERC CartographY Project
Senior Lecturer in Astrophysics
School of Physics and Astronomy
The University of Birmingham
Edgbaston
Birmingham  B15 2TT

Tel +44 (0) 121 414 4597
G.R.Davies@bham.ac.uk
grd349@gmail.com
davies@bison.ph.bham.ac.u <davies@bison.ph.bham.ac.uk>k
<davies@bison.ph.bham.ac.uk>

The contents of this e-mail may be privileged and are confidential. It may
not be disclosed, used, or copied in any way by anyone other than the
addressee. If received in error please notify the sender then delete it
from your system. Should you communicate with the sender by e-mail, you
consent to The University of Birmingham monitoring and reading any such
correspondence.

**Comment by Harry-Westwood** [comment]
@grd349 
First run on the mckeever grid with the correct delta_nu (including my previous method of increasing point density by interpolation not the method i suggested yesterday of increasing points). Setup: 64*5, no regularization, Nadam, elu and batch normalisation on the input layer.

I don't see any overfitting between tracks.
But the figure below shows that along the tracks the neural network deviates from the data
![image](https://user-images.githubusercontent.com/56080854/95198538-4f02a780-07d3-11eb-9d23-930f109541d5.png)
And the errors are much larger than i've seen for a long time.
![image](https://user-images.githubusercontent.com/56080854/95198662-7e191900-07d3-11eb-8e07-e9ff65243c67.png)
but the loss is 1x10-3 which is fine. 

I can't say whether this is down to the boundaries being too constrictive (4000 < Teff < 4800, 5 < L < 100, 3 < delta_nu < 10), the lack of data due reduction in available points due to the delta_nu calculations (total number of points in this grid is 70126). Or if i've messed something up and I haven't noticed during any of my checks. This could be the case because rather than remaking my NGC6791 RGB grid from the set of files I got from Tanda I matched the points in my grid to the points in the set of files on a point by point basis, though the reduction in points is roughly what I calculated it should be the reduction due to the change in delta_nu types. Though very occasionally a point from my grid couldn't be found in the set of files (which is strange) and I couldn't match roughly 10 of the tracks to any in the set of files, so I had to drop those tracks.

Could you make a suggestion on what i should do next? Be that create the grid again using my interpolation idea or just try some more neural network training configurations (i probably could train down the loss some more in the above neural network which might fix the deviation issues) or some other suggestion.
